### DISCUSSION TITLE
Use AIR for dual build for a go app with wasm

### DISCUSSION THREAD

**Discussion by larry868** [air-verse/air#370] [Q&A]
Hello everyone,

Building a web app with go and wasm require to build the server side with the traditional `go build` and also to build the front side with specific build `GOARCH=wasm GOOS=js go build -o web/app.wasm` to rebuild the web assembly code. Then the `wasm` built file is served by the server.

Source files taken into account are usually different but some can be shared for both builds.

How can I setup AIR to simultaneously auto-rebuild the server and live reload the generated app, and auto-rebuild the front ?

Thanks' for any help

**Comment by deanrie** [comment]
I'm interested too.
